docs/controls/TabbedCommandBar.md

```diff
@@ -0,0 +1,209 @@
+
+### TabViewItem Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Content | object | Main Tab Content. |
+| Header | object | Header Content of the Tab. |
+| HeaderTemplate | DataTemplate | Template for the Header object. |
+| Icon | IconElement | Icon of the Tab. |
+| IsClosable | bool | Set to true for the TabViewItem to show a close button. |
+
+## Events
+
+### TabView Events
+
+| Events | Description |
+| -- | -- |
+| TabClosing | Fires when a Tab is about to be closed, can be intercepted to prevent closure. |
+| TabDraggedOutside | Fires when a Tab is dragged outside of the Tab bar. |
+
+### TabViewItem Events
+| Events | Description |
+| -- | -- |
+| TabClosing | Fires when a Tab's closed button is clicked. |
+
+## Examples
+
+If you want to replicate the behavior of Microsoft Edge with the TabView, you can use the following setup:
+
+```xaml
+<controls:TabView TabWidthBehavior="Equal"
+                  CanCloseTabs="True"
+                  IsCloseButtonOverlay="True"
+                  CanDragItems="True"
+                  CanReorderItems="True"
+                  AllowDrop="True">
+  <controls:TabView.Resources>
+    <x:Double x:Key="TabViewItemHeaderMinHeight">32</x:Double>
+    <x:Double x:Key="TabViewItemHeaderMinWidth">90</x:Double>
+    <x:Double x:Key="TabViewItemHeaderMaxWidth">200</x:Double>
+  </controls:TabView.Resources>
+  ...
+</controls:TabView>
+```
+
+The TabView supports data binding as well.  The following example shows binding the TabView to a collection of 'DataItems' which have both 'Value' and 'MyText' properties:
+
+```xaml
+<controls:TabView x:Name="TabItems"
+                  ItemsSource="{Binding TabItemCollection}">
+  <controls:TabView.ItemHeaderTemplate>
+    <DataTemplate>
+      <TextBlock>
+        <Run Text="{Binding Value}"/>
+        <Run Text=": "/>
+        <Run Text="{Binding MyText}"/>
+      </TextBlock>
+    </DataTemplate>
+  </controls:TabView.ItemHeaderTemplate>
+  <controls:TabView.ItemTemplate>
+    <DataTemplate>
+      <StackPanel>
+        <TextBlock Margin="8">
+          <Run Text="Tab Value: "/>
+          <Run Text="{Binding Value}" />
+        </TextBlock>
+        <TextBlock Margin="8,0,0,0" Text="Some other shared content..." />
+      </StackPanel>
+    </DataTemplate>
+  </controls:TabView.ItemTemplate>
+</controls:TabView>
+```
+
+> [!NOTE]
+> It's recommended to use an [ObservableCollection](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1) when working with the TabView.
+
+## Sample Project
+
+[TabView Sample Page Source](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabView). You can [see this in action](uwpct://Controls?sample=TabView) in the [Windows Community Toolkit Sample App](http://aka.ms/uwptoolkitapp).
```

Comment:
The "see this in action" doesn't tell me what the link does or where it goes.

Try to be more descriptive about what the hyperlink is, avoid hyperlinking words like **here**, **this** or **it**. A more polished version would be

To see this in action, visit the [TabView demo](xxx) in the [Windows Community Toolkit Sample App](xxx).